lib/passageidentity/auth.rb

```diff
@@ -1,41 +1,53 @@
-require "openssl"
-require "base64"
-require "jwt"
-require_relative "client"
-require_relative "../openapi_client"
+# frozen_string_literal: true
+
+require 'openssl'
+require 'base64'
+require 'jwt'
+require_relative 'client'
+require_relative '../openapi_client'
 
 module Passage
+  # The Passage::Auth class provides methods for authenticating requests and tokens
+  # using the Passage authentication service. It supports fetching and caching
+  # JSON Web Keys (JWKS) for token verification, and provides methods for
+  # validating JWTs and revoking user refresh tokens.
```

Comment:
```suggestion
  # The Passage::Auth class provides methods for authenticating requests and tokens
  # using the Passage authentication service. It supports fetching and caching
  # JSON Web Keys (JWKS) for token verification, and provides methods for
  # validating JWTs and revoking user refresh tokens.
```

> using the Passage authentication service

the token validation actually happens locally (besides fetching the jwks) through a third-party lib, so it doesn't use our auth service

> It supports fetching and caching JSON Web Keys (JWKS) for token verification

i think this can be omitted since this should be private behavior, not exposed to the end user (even though it's currently is).